From a422c57f79d763f019ac00617d5801e66510c491 Mon Sep 17 00:00:00 2001
From: Huang Chaofan <chaofan.huang@nxp.com>
Date: Tue, 28 Aug 2018 14:47:56 +0800
Subject: [PATCH 4456/5242] MLK-19353 VPU: Rename VPU firmware binary

commit  5c188e72558cfb7069d54d5fe1962a232f84dd7b from
https://source.codeaurora.org/external/imx/linux-imx.git

Rename VPU firmware binary, change from 'vpu_fw_imx8qxp_xxx.bin' to
'vpu_fw_imx8_xxx.bin'

Signed-off-by: Huang Chaofan <chaofan.huang@nxp.com>
(cherry picked from commit d645e79676e178674e27c7706cf7080de5fdf3b8)
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 drivers/mxc/vpu-decoder-b0/vpu_b0.h         |    2 +-
 drivers/mxc/vpu-encoder-b0/vpu_encoder_b0.h |    2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/drivers/mxc/vpu-decoder-b0/vpu_b0.h b/drivers/mxc/vpu-decoder-b0/vpu_b0.h
index 5d7f305..86f4bd4 100755
--- a/drivers/mxc/vpu-decoder-b0/vpu_b0.h
+++ b/drivers/mxc/vpu-decoder-b0/vpu_b0.h
@@ -46,7 +46,7 @@
 
 #define VPU_MAX_FORMATS 4
 #define VPU_MAX_BUFFER 32
-#define M0FW_FILENAME "vpu/vpu_fw_imx8qxp_dec.bin"
+#define M0FW_FILENAME "vpu/vpu_fw_imx8_dec.bin"
 #define MMAP_BUF_TYPE_SHIFT 28
 #define MMAP_BUF_TYPE_MASK 0xF0000000
 #define M0_BOOT_SIZE 0x1000000
diff --git a/drivers/mxc/vpu-encoder-b0/vpu_encoder_b0.h b/drivers/mxc/vpu-encoder-b0/vpu_encoder_b0.h
index 9880d33..d726489 100644
--- a/drivers/mxc/vpu-encoder-b0/vpu_encoder_b0.h
+++ b/drivers/mxc/vpu-encoder-b0/vpu_encoder_b0.h
@@ -46,7 +46,7 @@
 
 #define VPU_MAX_FORMATS 4
 #define VPU_MAX_BUFFER 32
-#define M0FW_FILENAME "vpu/vpu_fw_imx8qxp_enc.bin"
+#define M0FW_FILENAME "vpu/vpu_fw_imx8_enc.bin"
 #define MMAP_BUF_TYPE_SHIFT 28
 #define MMAP_BUF_TYPE_MASK 0xF0000000
 #define M0_BOOT_SIZE 0x1000000
-- 
1.7.9.5

